lib/template/processors/svelte-esbuild-worker.js

```diff
@@ -0,0 +1,23 @@
+var esbuild = require('esbuild');
+var sveltePlugin = require('esbuild-svelte');
+
+function init() {
+  return function (options) {
+    var plugin = sveltePlugin({
+      // TODO: This will cause CSS to be generated that we don't need (the server already served it).
```

Comment:
Svelte just merged support for `css: 'none'` https://github.com/sveltejs/svelte/pull/7914